wagtail_wordpress_import/prefilters/handle_shortcodes.py

```diff
@@ -152,3 +152,13 @@ def construct_block(self, wagtail_custom_html):
                 "anchor_attrs": anchor_attrs,
             },
         }
+
+
+def filter_transform_shortcodes(html, options=None):
+    """
+    html: is the body content from one workpress item
+    options: not implemented
+    """
+    for handler in SHORTCODE_HANDLERS.values():
+        html = handler().pre_filter(html)
```

Comment:
**Question**: Having seen how this is used here, why are the shortcode handlers registered with a key, in the decorator? e.g. the decorator argument:

```
@register("caption")
class CaptionHandler(BlockShortcodeHandler):
    shortcode_name = "caption"
```

Reply:
It's a good point. This came over form Karl's earlier code example. I've made a lot of changes in the next PR and the following PR to capture the shortcode handler properties and it's possible I have used the key. If I change that here it might break going forward. I'll add an AC in ticket #70 capture a check on that if you are OK with that